oaknational/Oak-Web-Application
# refactor: about contact card from cms

## Description

- AboutContactCard gets data from cms

Headings will need to be set in portable text from the dropdown list in sanity to match designs - make sure there are no spaces between headings and text.

## Issue(s)

Progresses - About pages from CMS ticket

## How to test

1. Go to about pages - copy comes from sanity
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
